docs/qa/CheckLists/FavoritePage_tags.md

```diff
@@ -0,0 +1,68 @@
+| 19 | Below search there is statistics of how many tags and translations is presented                                                                                                                                                                          |             |                                                                                    |
+| 20 | To the right of a tag there is a "pencil" button,when pressed bottom sheet (14.3) appears to rename a tag                                                                                                                                                |             |                                                                                    |
+| 21 | All tags have a number to the right that indicates how many words have this tag                                                                                                                                                                          |             |                                                                                    |
+| 22 | Red button "create new tag" is moving down every time new tag is made and there is still space for it below                                                                                                                                              |             |                                                                                    |
+| 23 | If there are too many tags to show on 1 page, red button became fixed below and scroll option appears                                                                                                                                                    |             |                                                                                    |
+| 24 | Sorting option can be chosen by pressing "sort" button on top right corner", bottom sheet appears (14.1)                                                                                                                                                 |             |                                                                                    |
+|    | 1.4.1 sorting                                                                                                                                                                                                                                            |             |                                                                                    |
+| 25 | Image with stars that is similar to 14.1 frame                                                                                                                                                                                                           |             | Not sure about the image, no model currently. What happens when no result appears? |
```

Comment:
You mean if the bottom sheet with image and sorting filters appear when there's no tags, or what? If yes, we can just disable this sorting button